_schedule/tutorials/2018-10-14-09-00-t2-python-packaging-without-the-pain.md

```diff
@@ -0,0 +1,49 @@
+---
+abstract: Many of us have snippets of Python code we copy from project to project. Wouldn't it be nice if we could `pip install` them from PyPI instead of copy-pasting everywhere? But packaging has always been a complicated, scary task. We will take you from copy/paste to `pip install` in one tutorial!
+accepted: true
+category: tutorial
+date: 2016-07-18 09:00
+difficulty: Intermediate
+layout: session-details
+permalink: /tutorial/python-packaging-without-the-pain/
+presenters:
+- bio: "Diane Chen has 20+ years of experience programming Engineering CAD/CAM software, has built a few websites and worked in hardware testing.\r\nEnamored with the Python programming language, she teaches Python at [UC San Diego Extension](http://bit.ly/UCSDPython), workshops and private courses.
```

Comment:
Will that `\r\n` mess with anything?

Reply:
Not sure what it's intended to do.
![screen shot 2018-07-20 at 2 22 13 pm](https://user-images.githubusercontent.com/4193054/43025271-a9ebc2be-8c36-11e8-968c-c11702467743.png)

Reply:
Good eye! I caught most of them...